content/src/service/access/AccessCheckerImpl.ts

```diff
@@ -313,26 +314,13 @@ export class AccessCheckerImpl implements AccessChecker {
         query: string,
         variables: Record<string, any>
     ): Promise<T> {
-        const opts = {
-            method: 'POST',
-            headers: { 'Content-Type': 'application/json' },
-            body: JSON.stringify({ query, variables })
-        }
-
-        const res = await fetch(this.dclParcelAccessUrl, opts)
-        if (res.ok) {
-            const json = await res.json()
-            if (json.errors) {
-                throw new Error(
-                    `Error querying graph. Reasons: ${JSON.stringify(json.errors)}`
-                )
-            }
-            return json.data
+        const json = await this.fetcher.postForm(this.dclParcelAccessUrl, JSON.stringify({ query, variables }), { 'Content-Type': 'application/json' }, { attempts: 1 })
```

Comment:
Should we perform more than 1 attempt?

Reply:
If this fails, there is an external retries that has 5 attempts